shenyu-plugin/shenyu-plugin-brpc/src/main/java/org/apache/shenyu/plugin/brpc/proxy/BrpcProxyService.java

```diff
@@ -81,8 +81,7 @@ public Mono<Object> genericInvoker(final String body, final MetaData metaData, f
             }
         }
         initThreadPool();
-        CompletableFuture<Object> future = null;
-        future = new CompletableFuture<>().supplyAsync(() -> getValue(metaData, params), threadPool);
+        CompletableFuture<Object> future = CompletableFuture.supplyAsync(() -> getValue(metaData, params), threadPool);
```

Comment:
In the future, we use `com.baidu.cloud.starlight.api.rpc.threadpool.ThreadPoolFactory` SPI to impl it, this is a todo list.